frontend/src/css/Favorites.styl

```diff
@@ -0,0 +1,72 @@
+@import "~tracim_frontend_lib/src/css/Variable.styl"
+
+// NOTE - S.G. - 2021-04-01 - The positioning properties (flex/sizes) of the "columns"
+// should be kept in sync with ContentListItem.styl
+// Or a better solution (flex-grid?) be implemented.
+.favoritesHeader
+  display flex
+  width 100%
+  height 65px
+  justify-content space-between
+  align-items center
+
+  &__type
+    width 90px
+    text-align center
+  &__title
+    flex 2
+  &__title-max-sm
+    flex 2
+    display none
+  &__modification
+    flex 1
+  &__information
+    flex 1
+  &__favoriteButton
+    text-align center
+    width 90px
+
+.contentListItem
+  &__favoriteButton
+    font-size fontSizeTitle
+    text-align center
+    width 90px
+  &__link
```

Comment:
if this rule is an override, it should only apply to the favorite page context
if it's not an override, I think it should be in contentListItem.styl

Reply:
Done.